src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -141,7 +141,7 @@ class ValidationStep extends React.Component {
                                 {this.props.translate('validationStep.descriptionCTA')}
                             </Text>
                         </View>
-                        <View style={[styles.m5, styles.flex1]}>
+                        <View style={[styles.m5]}>
```

Comment:
Also feels like this should be a `ScrollView` for consistency with the other forms.

Reply:
Thanks you for request. I set ScrollView at line 135 instead of 144. Because above the line 144, some information is displayed in a sibling View at line 136-143 (validationStep.description and validationStep.descriptionCTA) It is also part of this page/form, So view at line 135 is parent of both ( i.e. view at 136, and 144) so I put ScrollView at parent level view i.e. at line 135 So form instruction and input field both comes within ScrollView and scroll both together and page behave normally.